### PR TITLE
Menu test

### PR DESCRIPTION
**Pull request by PaulRoh97**
Made tests for Menu items. Menu item cards use opacity (CSS), so a visual check may be required as well.

### CODE REVIEW

**Comment by PaulRoh97** [comment]
I think this is somehow looking up all of our changes since the past few days. I am not sure why. The only files I changed were index.njk and menu.test.js

File: Covid Lunches/express/express-bootstrap/models/index.js [hunk 1]
```diff
@@ -14,6 +14,6 @@ db.Sequelize = Sequelize;
 db.sequelize = sequelize;
 
 db.tutorials = require("./tutorial.model.js")(sequelize, Sequelize);
-db.accounts = require("./account.js")(sequelize, Sequelize);
+db.accounts = require("./account.model.js")(sequelize, Sequelize);
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/models/index.js:17]: I'm guessing we are formalizing how our use account in organized.

File: Covid Lunches/express/express-bootstrap/public/js/__test__/menu.test.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const puppeteer = require('puppeteer')
+
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/public/js/__test__/menu.test.js:2]: All tests passed on suite run.

File: Covid Lunches/express/express-bootstrap/public/js/login.js [hunk 3]
```diff
@@ -1,97 +1,77 @@
-// get the modal
-let modal = document.getElementById("modal");
-// get the element that is not the login form
-let container = document.getElementById("container");
-// get the button that opens the login form
-let loginBtn = document.getElementById("login-btn");
-// grab the login button
-let loginFormBtn = document.getElementById('login-form-btn');
+const NEW_PASS = 0,
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/public/js/login.js:1]: Probably good idea to have nice constants all the way at the beginning.

File: Covid Lunches/express/express-bootstrap/public/js/login.js [hunk 4]
```diff
@@ -1,97 +1,77 @@
...
-window.onclick = function (event) {
-    if (event.target == modal) {
-        modal.style.display = "none";
-        document.getElementById('email').value = '';
-        document.getElementById('password').value = '';
-        clearError('email-alert');
-        clearError('pass-alert');
-    }
-}
+document.querySelector('.sign-in-btn').addEventListener('click', (event) => {
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/public/js/login.js:7]: YES! DATABASE USE!

File: Covid Lunches/express/express-bootstrap/public/js/login.js [hunk 5]
```diff
@@ -1,97 +1,77 @@
...
+        // get elements
+        const inputField = document.querySelector(`#${childElement}`)
+        // create text node and append to div
+        errorDiv.appendChild(document.createTextNode(errorMessage))
+        errorDiv.style.color = 'red'
+        errorDiv.style.paddingLeft = '5px'
+        errorDiv.style.paddingRight = '5px'
+        errorDiv.style.marginBottom = '20px'
+        errorDiv.style.borderRadius = '5px'
+        // insert error message above input field
```
Comment (F-421) [inline Covid Lunches/express/express-bootstrap/public/js/login.js:67]: I didn't know you could style something like that. Cool!

**Review by F-421 — APPROVED** [review]
I like it. Approved.

**Review by rucbarr — APPROVED** [review]
Thumbs up.

**Pull request merged**